### PR TITLE
Renamed to pyviz_comms and added Python package

### PR DESCRIPTION
**Pull request by jlstevens**
This PR add the shared Python comms component used by pyviz projects. This complements the Jupyterlab extension which must be installed to allow comms to work in that environment.

### CODE REVIEW

File: conda.recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - pyviz_comms
+
+about:
+  home: www.holoviews.org
```
Comment (philippjfr) [inline conda.recipe/meta.yaml]: Point to pyviz.org?

File: pyviz_comms/__init__.py [hunk 2]
```diff
@@ -0,0 +1,435 @@
...
+    from StringIO import StringIO
+except:
+    from io import StringIO
+
+
+
+PYVIZ_PROXY = """
+if (window.PyViz === undefined) {
+   if (window.HoloViews === undefined) {
+     var PyViz = {comms: {}, comm_status:{}, kernels:{}, receivers: {}, plot_index: []}
```
Comment (philippjfr) [inline pyviz_comms/__init__.py]: Consistent spacing would be nice if you're already making the change above.

Comment (jlstevens) [reply pyviz_comms/__init__.py]: Fixed in e9f2157

File: conda.recipe/meta.yaml (hunk 1 above)
Comment (jlstevens) [reply conda.recipe/meta.yaml]: Addressed in 7e1ef97

**Comment by jlstevens** [comment]
If we are happy with the current setup.py, I'll register this on PyPI and then this PR can be merged.

**Comment by jlstevens** [comment]
@philippjfr @jbednar Probably makes more sense to merge this and then register on PyPI.

**Pull request merged**